dts/common/nordic/nrf9280.dtsi

```diff
@@ -0,0 +1,1207 @@
+			};
+
+			spi135: spi@9c6000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x9c6000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <454 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				rx-delay-supported;
+				rx-delay = <1>;
+			};
+
+			uart135: uart@9c6000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x9c6000 0x1000>;
+				status = "disabled";
+				interrupts = <454 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			dppic136: dppic@9d1000 {
+				compatible = "nordic,nrf-dppic-global";
+				reg = <0x9d1000 0x1000>;
+				status = "disabled";
+			};
+
+			timer136: timer@9d2000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x9d2000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <466 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				prescaler = <0>;
+			};
+
+			timer137: timer@9d3000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x9d3000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <467 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				prescaler = <0>;
+			};
+
+			pwm133: pwm@9d4000 {
+				compatible = "nordic,nrf-pwm";
+				reg = <0x9d4000 0x1000>;
+				status = "disabled";
+				interrupts = <468 NRF_DEFAULT_IRQ_PRIORITY>;
+				#pwm-cells = <3>;
+			};
+
+			i2c136: i2c@9d5000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x9d5000 0x1000>;
+				status = "disabled";
+				interrupts = <469 NRF_DEFAULT_IRQ_PRIORITY>;
+				easydma-maxcnt-bits = <15>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			spi136: spi@9d5000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x9d5000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <469 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				rx-delay-supported;
+				rx-delay = <1>;
+			};
+
+			uart136: uart@9d5000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x9d5000 0x1000>;
+				status = "disabled";
+				interrupts = <469 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			i2c137: i2c@9d6000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x9d6000 0x1000>;
+				status = "disabled";
+				interrupts = <470 NRF_DEFAULT_IRQ_PRIORITY>;
+				easydma-maxcnt-bits = <15>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			spi137: spi@9d6000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x9d6000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <470 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				rx-delay-supported;
+				rx-delay = <1>;
+			};
+
+			uart137: uart@9d6000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x9d6000 0x1000>;
+				status = "disabled";
+				interrupts = <470 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+		};
+	};
+
+	cpuapp_ppb: cpuapp-ppb-bus {
```

Comment:
> Why this node is present in a common file instead of in a cpuapp specific file? I don't think any other CPU should have it included in their device trees.

The private bus nodes were introduced to help keep the entire SoC description in one file.

Reply:
What is the point of the entire SoC description in one file? I don't think any other CPU is interested in App having something at its PPB.

Private peripherals' nodes in the device tree files are unlike the common memory description. The value of the common memory map makes it easier to avoid mistakes when repartitioning the system (replacement of missing system device tree).

Reply:
The `<soc>.dtsi` is included by every CPU, but anything that shouldn't be visible is masked out using `/delete-node/`. Think of each `<soc>_<cpu>.dtsi` as a projection of `<soc>.dtsi`, which mostly subtracts content and doesn't add too much new stuff. The point is to organize a relatively complex SoC as tightly as possible, in as few files as possible. It's also inspired by would-be operations on a system devicetree.

Reply:
It's confusing for me to create cpu-specific nodes in common files only to `delete-node` them later in cpu-specific files. (but at least I see the reason for these long `/delete-node/` lists I've seen recently so many times)

The system device tree would be another story, but I guess it would generate the CPU-specific device tree files containing only nodes related to the target CPU. So I imagined we aim to mimic the result of the system device tree tooling.

It's not a strong opinion. I see your point. The fact that it makes reading the files harder for me does not mean the result is the same for everyone.

Reply:
> The fact that it makes reading the files harder for me does not mean the result is the same for everyone.

I can see that it can be difficult to get used to. This approach has only existed in-tree for a few months, so it's good to re-examine it every once in a while :)

Reply:
Ok, let's keep it unified for now and we can open discussion to change it globally for all Nordic .dts files in another PR